pkg/collector/k8s_api.go

```diff
@@ -72,14 +72,17 @@ func NewK8sAPICollector(ctx context.Context, cfg *config.KubehoundConfig) (Colle
 	if err != nil {
 		return nil, err
 	}
+	if clusterName == "" {
+		return nil, errors.New("Cluster name is empty. Did you forget to set `KUBECONFIG` or use `kubectx` to select a cluster?")
+	}
```

Comment:
Did you ever got in this situation ? (wondering)

Reply:
I did when testing a few thing starting from my personal PC that never connected to k8s.